physics/sfcsub.F

```diff
@@ -7491,9 +7491,6 @@ subroutine clima(lugb,iy,im,id,ih,fh,len,lsoil,slmskl,slmskw,     &
           endif
           call abort
         endif
-!
-!  soil type
-        print *,'in FIXREAD fnsotc =',fnsotc 
```

Comment:
@HelinWei-NOAA is this the print that produces all the output in the log file? I thought that was related to soil type. Did that ever get removed?

Reply:
Yes. fnsotc is the namelist parameter used to point to the soil type file